napari/_qt/qt_main_window.py

```diff
@@ -915,7 +915,7 @@ def _add_menus(self):
         )
         self.main_menu.addMenu(self.plugins_menu)
         # debug menu (optional)
-        if perf.USE_PERFMON is not None:
```

Comment:
Just want to ping @Czaki who made the original suggestion: https://github.com/napari/napari/pull/6915#discussion_r1627115444

From [docs](https://napari.org/dev/howtos/perfmon.html), I am guessing we set `NAPARI_PERFMON` to be 0 or 1, so I think `is not None` would make it 'always' on but please let us know if we've missed something @Czaki

Reply:
Just in case, I concluded `perf.USE_PERFMON` being boolean from its definition at https://github.com/napari/napari/blob/b3a8dd22895c913d8183735f52b9d1d71c963d7f/napari/utils/perf/__init__.py#L64

Checking the original suggestion I think it refers to `perf.perf_config`. Maybe that is a different way to check for perfmon activation using a `None`/`not None` definition? 🤔

Reply:
I see `perf.perf_config` gets it's value from:

https://github.com/napari/napari/blob/eab7661459e70479c7c7d587a36463f3b099b64a/napari/utils/perf/_config.py#L160-L168

Maybe the better option is to change the code to:

```python
if perf.perf_config is not None:
```
?

@Czaki ?

Reply:
I think that both options are valid to resolve this problem. But in general, we should select one and update the whole codebase to it. As `perf_config` is wider used, maybe `perf.perf_config is not None` is better?

Reply:
+1 agreed.

We seem to do things different in debug vs rest of codebase: https://github.com/napari/napari/pull/6975#discussion_r1637580041

Reply:
Just to be sure, should I change only this line to be `perf.perf_config is not None` and over a follow up PR make the whole codebase use only `perf.perf_config is not None`? Or maybe it could be better to make that whole codebase change on this PR?

Reply:
I think we can change the rest in a new PR, it's cleaner